commit e8db58e3d80bf4c739450a3f0791022bde391cb4
Author: Geert Uytterhoeven <geert+renesas@glider.be>
Date:   Thu Jun 14 15:48:08 2018 +0200

    arm64: dts: renesas: salvator-common: Fix adv7482 decimal unit addresses
    
    [ Upstream commit c5a884838ce34681200b5a45b2330177036affd0 ]
    
    With recent dtc and W=1:
    
        ...salvator-x.dtb: Warning (graph_port): /soc/i2c@e66d8000/video-receiver@70/port@10: graph node unit address error, expected "a"
        ...salvator-x.dtb: Warning (graph_port): /soc/i2c@e66d8000/video-receiver@70/port@11: graph node unit address error, expected "b"
    
    Unit addresses are always hexadecimal (without prefix), while the bases
    of reg property values depend on their prefixes.
    
    Fixes: 908001d778eba06e ("arm64: dts: renesas: salvator-common: Add ADV7482 support")
    Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
    Reviewed-by: Rob Herring <robh@kernel.org>
    Acked-by: Kieran Bingham <kieran.bingham+renesas@ideasonboard.com>
    Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/arch/arm64/boot/dts/renesas/salvator-common.dtsi b/arch/arm64/boot/dts/renesas/salvator-common.dtsi
index 9256fbaaab7f..5853f5177b4b 100644
--- a/arch/arm64/boot/dts/renesas/salvator-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/salvator-common.dtsi
@@ -440,7 +440,7 @@
 			};
 		};
 
-		port@10 {
+		port@a {
 			reg = <10>;
 
 			adv7482_txa: endpoint {
@@ -450,7 +450,7 @@
 			};
 		};
 
-		port@11 {
+		port@b {
 			reg = <11>;
 
 			adv7482_txb: endpoint {
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1164_e8db58e3d80bf4c739450a3f0791022bde391cb4.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1164_e8db58e3d80bf4c739450a3f0791022bde391cb4.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
